Apps/Src/Task_VoltTempMonitor.c

```diff
@@ -65,7 +79,14 @@ void Task_VoltTempMonitor(void *p_arg) {
         // BLOCKING =====================
         // Update Temperature Measurements
         Temperature_UpdateAllMeasurements();
-        
+        CanMsg.id = TEMP_DATA;
+        for (int i = 0; i < NUM_BATTERY_MODULES; i++){ //send all battery module temp data
+            CanPayload.idx = i;
+            CanData.f = Temperature_GetModuleTemperature(i);
```

Comment:
This function returns an `int32_t`

Reply:
How should I convert this to a float? I'm not sure how the data is being read (are all 32 bits meant to be for 2 digit celsius numbers)?